packages/vitest-example-test-suite/src/excludedTest.spec.ts

```diff
@@ -0,0 +1,7 @@
+import { expect, test } from "vitest";
+import { add } from "./add";
+
+test("adds 1 + 2 to equal 3", async () => {
+  await new Promise((resolve) => setTimeout(resolve, 5000));
+  expect(add(1, 2)).toBe(3);
+}, 10000);
```

Comment:
Let's add this since we don't have tests verifying that the Vitest config is respected by Knapsack Pro. If the excluded test ever runs unintended, we will know because the CI will fail.

```suggestion
}, 10000);

test("excluded test should not be run (verify vitest.config.ts is respected)", () => {
  expect(true).toBe(false);
});
```